.gitignore

```diff
@@ -35,6 +35,9 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
+/.angulardoc.json
+/s4e-web/dist/
```

Comment:
I'd move it to /s4e-web/.gitignore

Reply:
.angulardoc.json is generated file by VS depend on opened folder, so I'll keep it in here. dist folder I'll repair in global error PR, so this changes can be build and merged right now